### DISCUSSION TITLE
interpolation method embedded in interp_like()

### DISCUSSION THREAD

**Discussion by MahdiMontazeri** [DHI/mikeio#708] [Q&A] [answered]
what is the default interpolation method in interp_like function? is there a way to change the interpolation method?

**Comment by ecomodeller** [comment, marked as answer]
The interpolation method available is IDW (Inverse Distance Weighted) https://github.com/DHI/mikeio/blob/97872ab01bf47c37121d75a769e0edaf7cf8afaa/mikeio/_interpolation.py#L11
